### PR TITLE
Prometheus, AMP, and AMG Website Guide

### PR DESCRIPTION
**Pull request by BriannaRoskind**
**Which issue is resolved by this Pull Request:**
Adds a guide explaining how to setup Prometheus and Amazon Managed Service for Prometheus.
There are also links to the Amazon Managed Grafana documentation explaining how to perform that portion of the setup.

**Description of your changes:**
Added a guide in the website/content/en/docs/deployment/add-ons/prometheus folder.

### CODE REVIEW

**Review by rrrkharse — APPROVED** [review]

File: deployments/add-ons/prometheus/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    3. Make sure to replace the following in the above commands:
+        * **\<your-cluster-name\>**
+        * **\<your-cluster-region\>**
+2. Create an IAM Policy:
+    1. `export AMP_POLICY_NAME=<desired-amp-policy-name>`
+    2. Make sure to replace the following in the above command:
+        * **\<desired-amp-policy-name\>** - a policy name of your choosing
+    3. `export AMP_POLICY_ARN=$(aws iam create-policy --policy-name $AMP_POLICY_NAME --policy-document file://deployments/add-ons/prometheus/AMPIngestPermissionPolicy.json --query 'Policy.Arn' | tr -d '"')`
+3. Create a Service Account:
+    1. `eksctl create iamserviceaccount --name amp-iamproxy-ingest-service-account --namespace monitoring --cluster $CLUSTER_NAME --attach-policy-arn $POLICY_ARN --override-existing-serviceaccounts --approve —-region $CLUSTER_REGION`
```
Comment (rrrkharse) [inline deployments/add-ons/prometheus/README.md:22]: $POLICY_ARN --> $AMP_POLICY_ARN

File: deployments/add-ons/prometheus/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Many Kubeflow users utilize Prometheus and Grafana to monitor and visualize their metrics. However it can be difficult to scale open source Prometheus and Grafana as the number of nodes to be monitored increases. AMP seeks to simplify this issue by allowing multiple Prometheus servers to aggregate their metrics in Amazon Managed Prometheus, and finally Amazon Managed Grafana allows customers to then view these aggregated metrics.
+
+## Prerequisites
+First download one of our deployment options by following the directions at: https://awslabs.github.io/kubeflow-manifests/docs/deployment/prerequisites/
+
+**Note:** The steps below will assume you are sitting in the kubeflow-manifests directory.
+
+## Steps to Setup Prometheus and AMP
+1. Export cluster name and region as environment variables:
+    1. `export CLUSTER_NAME=<your-cluster-name>`
```
Comment (rrrkharse) [inline deployments/add-ons/prometheus/README.md:11]: Can we convert the inline code to code blocks:

E.g.
```
export CLUSTER_NAME=<your-cluster-name>
```

I think the inline blocks mess up the `--` dashes which makes the commands fail when you copy paste.

File: deployments/add-ons/prometheus/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Steps to Verify Prometheus and AMP are Connected
+1. Get the Prometheus Pod name:
+    1. `export PROMETHEUS_POD_NAME=$(kubectl get pods --namespace=monitoring | grep "prometheus-deployment" | cut -d' ' -f1)`
+2. Start port-forwarding for the prometheus pod:
+    1. `kubectl port-forward $PROMETHEUS_POD_NAME 9090:9090 --namespace=monitoring &`
+3. Navigate to the tests directory inside kubeflow-manifests:
+    1. `cd tests`
+5. Run the below command to verify the KFP create experiment count metric is being correctly exported to AMP:
+    1. `python3 -c 'import os; import e2e.utils.prometheus.setup_prometheus_server as setup_prometheus_server; cluster_region = os.environ["CLUSTER_REGION"]; workspace_id = os.environ["AMP_WORKSPACE_ID"]; setup_prometheus_server.check_AMP_connects_to_prometheus(cluster_region, workspace_id, expected_value=0)'`
```
Comment (rrrkharse) [inline deployments/add-ons/prometheus/README.md:53]:
```
    access_key = os.environ['AWS_ACCESS_KEY_ID']
    secret_key = os.environ['AWS_SECRET_ACCESS_KEY']
```

These need to be exported for the script to work. When testing locally I had to export those variables.

Comment (rrrkharse) [reply deployments/add-ons/prometheus/README.md:53]: This step also requires `awscurl` to be installed. We need to add a link in the prerequisites to download `awscurl`.

**Review by rrrkharse — CHANGES_REQUESTED** [review]

**Comment by rrrkharse** [comment]
Could you also move the readme to this folder:
https://github.com/awslabs/kubeflow-manifests/tree/main/website/content/en/docs/deployment/add-ons

**Review by rrrkharse — APPROVED** [review]

**Review by ryansteakley — APPROVED** [review]

**Pull request merged**